release-schema.json

```diff
@@ -104,6 +104,25 @@
           "openCodelist": false,
           "uniqueItems": true,
           "minItems": 1
+        },
+        "forReduction": {
+          "title": "For reduction?",
+          "description": "A true/false field to indicate if the criteria (or criterion) will (only) be used to select the candidates to be invited for the second stage of the procedure (if a maximum number of candidates was set).",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "numbers": {
+          "title": "Selection criterion numbers",
+          "description": "Information about the numbers linked to the selection criteria.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AwardCriterionNumber"
```

Comment:
If we want to reuse the `AwardCriterionNumber` definition, I think we'll need to update the code descriptions in `criterionThreshold.csv` to cover requests to participate (currently, they only mention bids) and we should probably rename the definition to `CriterionNumber`, too.

Reply:
Agreed, I'll create a new issue in the extensions repo for this. https://github.com/open-contracting/ocds-extensions/issues/199

Reply:
I think that we should make those changes to the award criteria extension before merging this PR, to avoid this extension being in an incoherent state.

Reply:
Perhaps worth adding it to the [tracker](https://docs.google.com/spreadsheets/d/1VIAFbuTCF1kMhP7ngWnlFrE9HXSUb6I6W8emlkUy3K8/edit#gid=1619313430) with a note, since the changes relate to eForms.